games-roguelike/hengband/hengband-1.6.2-r1.ebuild

```diff
@@ -14,7 +14,7 @@ LICENSE="Moria"
 SLOT="0"
 IUSE="X l10n_ja"
 
-RDEPEND=">=sys-libs/ncurses-5:0
+RDEPEND=">=sys-libs/ncurses-5:0[tinfo]
```

Comment:
This kind of works, although I'd prefer to not force this. Rebuilding ncurses with or without tinfo breaks ABI and it's kind of a pain point. If someone has it off, they may have a good reason (like some esoteric Prefix case or similar).

The solution is linked to my other comment coming...

Reply:
Removed it per our IRC convo.